packages/server/graphql/public/types/AddReactjiToReactableSuccess.ts

```diff
@@ -0,0 +1,20 @@
+import {ReactableEnumType} from '../../types/ReactableEnum'
+import {AddReactjiToReactableSuccessResolvers} from '../../public/resolverTypes'
+
+export type AddReactjiToReactableSuccessSource = {
+  reactableId: string
+  reactableType: ReactableEnumType
+  addedKudosId?: number | null
+}
+const AddReactjiToReactableSuccess: AddReactjiToReactableSuccessResolvers = {
+  reactable: async ({reactableId, reactableType}, _args: unknown, {dataLoader}) => {
+    // TODO: refactor to return specific type
+    return (await dataLoader.get('reactables').load({id: reactableId, type: reactableType})) as any
```

Comment:
-1 You just need to tell GraphQL what type it needs to expect here by adding
```
          "Reactable": "../../database/types/Reactable#Reactable",
```
to the mappers in `codegen.json`